test/cookbooks/mysql_tuning_test/recipes/ohai_plugin.rb

```diff
@@ -19,5 +19,11 @@
 # limitations under the License.
 #
 
+# Silence extraneous stderr from ohai
+# "INFO: The plugin path /etc/chef/ohai/plugins does not exist. Skipping..."
+directory '/etc/chef/ohai/plugins' do
+  recursive true
+end
```

Comment:
I'm not sure that we should. It's very much a workaround for tests only. Under Test Kitchen, Ohai's plugin directory is /tmp/kitchen/ohai/plugins/, but Ohai still erroneously emits stderr `"INFO: The plugin path /etc/chef/ohai/plugins does not exist. Skipping..."` because it assumes that path (https://github.com/chef/ohai/blob/7254fff3cbcc4ea489259c7d7a86e9c317148bdf/lib/ohai/config.rb#L37)

Even setting the following in .kitchen.yml and passing the `ohai` command the client.rb config doesn't fix it.
```
provisioner:
  name: chef_zero
  client_rb:
    ohai.directory: /tmp/kitchen/ohai/plugins
```

If anything, I think it would need to be addressed upstream in https://github.com/chef/ohai/.